jvm-runtime/ftl-runtime/kotlin/build-parent/pom.xml

```diff
@@ -28,6 +31,26 @@
                 <artifactId>kotlin-maven-plugin</artifactId>
                 <version>${kotlin.version}</version>
                 <executions>
+                    <execution>
+                        <id>kapt</id>
+                        <goals>
+                            <goal>kapt</goal>
+                        </goals>
+                        <configuration>
+                            <sourceDirs>
+                                <sourceDir>src/main/kotlin</sourceDir>
+                                <sourceDir>src/main/java</sourceDir>
+                            </sourceDirs>
+                            <annotationProcessorPaths>
+                                <!-- Specify your annotation processors here. -->
+                                <annotationProcessorPath>
+                                    <groupId>xyz.block.ftl</groupId>
+                                    <artifactId>ftl-java-runtime</artifactId>
+                                    <version>${project.version}</version>
```

Comment:
Is this running AnnotationProcessor in the Kotlin build?

Reply:
Yes, it turns out that this is supported. Kotlin 2.0 is bringing in a new approach but it seems a bit immature at the moment. 

Thinking about it I guess I should move the processor to common.